zookeeper-server/src/main/java/org/apache/zookeeper/server/ZooKeeperServer.java

```diff
@@ -1358,6 +1395,85 @@ static void setMaxBatchSize(int size) {
         maxBatchSize = size;
     }
 
+    private void initLargeRequestThrottlingSettings() {
+        setLargeRequestMaxBytes(Integer.getInteger("zookeeper.largeRequestMaxBytes", 100 * 1024 * 1024));
```

Comment:
Why not just use the already defined largeRequestMaxBytes int here for default value?

Reply:
Good question and I don't know an answer :) changed to use largeRequestMaxBytes instead